### PR TITLE
changed mail to anders.

### PR DESCRIPTION
**Pull request by strobba**
Solves issue #74.

### CODE REVIEW

**Comment by docNord** [comment]
I am of a mind that info@ingenjorsarbeteforklimatet.se is more generic and leaves the opportunity for someone other than me can keep the correspondance going, what do you think?

**Comment by docNord** [comment]
Added solution to have this PR also closes #79.

I still think we should direct e-mails to info@ingenjorsarbeteforklimatet.se though. How can we find a way to address this in such a way that we may close the PR?

File: content/pages/kontakt.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ status: published
 Kan vi hjälpa dig att hjälpa klimatet gör vi gärna det. Vi har mångårig
 erfarenhet inom ingenjörskonst och en passion för klimatarbete.
 Eller kanske du vill bli en av oss och bidra och hjälpa till i klimatarbetet,
-kontakta oss då på sociala medier eller skicka ett mail till [info@ingenjorsarbeteforklimatet.se](mailto:info@ingenjorsarbeteforklimatet.se)
+kontakta oss då på sociala medier eller skicka ett mail till [anders.nord@ingenjorsarbeteforklimatet.se](mailto:anders.nord@ingenjorsarbeteforklimatet.se)
```
Comment (docNord) [inline content/pages/kontakt.md:12]: I think info@ingenjorsarbeteforklimatet.se looks better here. Couldn't we forward the emails from that mail account to us three, @strobba @mgcth and @docNord and maintain the same address?

File: content/posts/ifk-smhi-2023-03-29.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ har föreningen, främst genom dr. Mladen Gibanicas försorg, arbetat med att ta
 fram ett pythonbibliotek för att på ett enkelt sätt läsa data från SMHIs API.
 Söndagen den 27 mars släpptes den första officiella versionen, och nu finns
 version 0.1.1 tillgänglig via paketinstallationsplattformen
-<a href=""https://pypi.org/project/ifk-smhi/" target="_blank">pypi</a>.
+<a href=""<https://pypi.org/project/ifk-smhi/>" target="_blank">pypi</a>.
```
Comment (docNord) [inline content/posts/ifk-smhi-2023-03-29.md:18]: Don't know what happened here, any suggestions @mgcth ?

**Review by docNord — APPROVED** [review]
Just one thing I'd like @mgcth to comment on and possibly have me fix before we merge.

File: content/posts/ifk-smhi-2023-03-29.md (hunk 2 above)
Comment (mgcth) [reply content/posts/ifk-smhi-2023-03-29.md:18]: I think you should remove <> and the extra ".

**Review by mgcth — COMMENTED** [review]
Are these all places your email is in?

**Comment by docNord** [comment]
> Are these all places your email is in?

I think so, yes. I went through all md files and replaced it with.

**Pull request merged**